src/rhsmlib/client_info.py

```diff
@@ -13,8 +13,8 @@
 
 """
 This module holds information about current state of client application
-like sender of D-bus method, current subscription-manager command (register,
```

Comment:
Removed `register` too, so it’s not the only example. I was not sure what other command to put there instead of `attach`. There is no example of a DNF command too.

Reply:
keeping "register", even as the only example should be fine

Reply:
I put a reference to the `status` command there. It sounds like the best command example after `register`.